options/options.go

```diff
@@ -28,6 +28,9 @@ const DEFAULT_PARALLELISM = math.MaxInt32
 // TERRAFORM_DEFAULT_PATH just takes terraform from the path
 const TERRAFORM_DEFAULT_PATH = "terraform"
 
+// DEFAULT_LOG_LEVEL defines default log level for Terragrunt
+const DEFAULT_LOG_LEVEL = util.DEFAULT_LOG_LEVEL
```

Comment:
NIT: Why create an alias rather than use `util.DEFAULT_LOG_LEVEL` everywhere?

Reply:
Yeah, I was not sure about it. I did the alias because all our options are in `options` package. So like we are setting `DEFAULT_PARALLELISM = math.MaxInt32` based on `math` module, here we set defaults based on `util` package. We can use `util` though, but then options would become scattered across different packages. What do you think?

Reply:
I don't feel strongly either way, so OK to leave as is.